### ISSUE TITLE
Proof of concept to validate that flexbench can be wrapped as a desktop app

### ISSUE THREAD

**Issue by iskitsas** [flexivian/flexbench#19]
Implement a simple desktop app, having just one button, when clicked running a simple flexbench scenario, showing the results to the user. 

This poc will validate that the flexbench nodejs lib can be integrated in a desktop app, anyone could install at his PC

**Comment by iskitsas** [comment]
Agreed to have a look on 

- https://docs.nodegui.org/
- https://react.nodegui.org/

**Comment by Kanha-13** [comment]
While researching more on `nodegui/react-nodegui` I came to the conclusion that its not yet ready for the complex desktop app. Even in their github https://github.com/nodegui/react-nodegui readme.md its mentioned that it is not ready for any complex apps, only suitable for small simple apps. So I guess `electronjs` will be good to go tech stack for building wrapper for this library.
Although I have made POC using both tech stacks. 

**But there is a huge difference in the app size**

- ElectronJs around 550MB
- NodeGUi-React 180MB

Waiting for your thoughts on this! @iskitsas
